.github/workflows/ready-for-review.yml

```diff
@@ -1,15 +0,0 @@
-name: "Assign Reviewers"
```

Comment:
Why did you remove it?

Reply:
Please take a look at the ticket.

Reply:
it doesn't work: https://github.com/rowi1de/auto-assign-review-teams/issues/17#issuecomment-967261575